src/_data/sponsors.yml

```diff
@@ -1,9 +1,20 @@
-# Uncomment to add the sponsors
-#
-#- title: Level Name
-#  slug: level-name
-#  companies:
-#    - name: Sponsor Name
-#      url: https://www.sponsor.com/
-#      logo_url: /assets/images/pages/home/sponsors/sponsor.svg
-#      logo_style: height:64px;
+- name: AppSignal
+  about: "APM for Ruby, Elixir & Node.js. No-brainer monitoring for smart developers."
+  logo_url: /images/sponsors/appsignal.svg
+  url: "https://www.appsignal.com/"
+- name: Eventpop
+  about: "Eventpop is an end-to-end event solution that enhances the seamless event experience. Eventpop แพลตฟอร์มจำหน่ายบัตรที่ตอบโจทย์ทุกงานอีเวนต์."
+  logo_url: /images/sponsors/eventpop.png
+  url: "https://www.eventpop.me/"
+- name: Honeybadger
+  about: "Application Health Monitoring built for Developers. Everything you need to keep production happy so that you can keep shipping. Deploy with confidence and be your team's DevOps hero."
+  logo_url: /images/sponsors/honeybadger.png
+  url: "https://www.honeybadger.io/"
+- name: Oivan
+  about: "We have local teams in 7 cities in 5 countries. With our proprietary Enhanced Distributed Teams model, our clients benefit from close onsite collaboration and the scale of our internationally distributed team."
+  logo_url: /images/sponsors/oivan.png
+  url: "hhttps://oivan.com/"
```

Comment:
There's an extra `h` at the beginning of the URL.

Reply:
Good eyes 🕵️ Fixed in 31f3840